spring-boot/src/main/java/org/springframework/boot/context/embedded/undertow/UndertowEmbeddedServletContainerFactory.java

```diff
@@ -345,6 +362,29 @@ private DeploymentManager createDeploymentManager(
 		return manager;
 	}
 
+	private void configureAccessLog(DeploymentInfo deploymentInfo) {
+		deploymentInfo.addInitialHandlerChainWrapper(new HandlerWrapper() {
+			@Override
+			public HttpHandler wrap(HttpHandler handler) {
+				try {
+					Xnio xnio = Xnio.getInstance(Undertow.class.getClassLoader());
+					XnioWorker worker = xnio.createWorker(OptionMap.builder().getMap());
+					File baseDir = (baseDirectory != null ? baseDirectory
```

Comment:
Looks good. I'm not totally sure how useful it is to log to a temporary directory, but Tomcat has the same issue so this isn't the place to worry about that.